backend/util/wordhint/wordhint.go

```diff
@@ -0,0 +1,56 @@
+package wordhint
+
+import (
+	"math/rand"
+	"time"
+)
+
+type WordHint struct {
+	previousHintIndexes []int
+}
+
+func randInt(min int, max int) int {
+	rand.Seed(time.Now().UTC().UnixNano())
+	return min + rand.Intn(max-min)
```

Comment:
You should make a Get function, and inside that function, you can initialize the randInt seed that way. 
```go

// I need this so I can mock your utility
type WordHint interface {
	GiveHint(word string) string
}

// make this lowercase now
type wordHint struct {
	previousHintIndexes []int
}
func Get() *wordHint {
    rand.Seed(time.Now().UTC().UnixNano())
    return &wordHint {}
}
```